views/article.html

```diff
@@ -47,7 +47,7 @@ <h2 class="article-standfirst">{{standfirst}}</h2>
 </article>
 
 <div class="comments">
-	{{> link class='comments-button' href=(concat canonicalURL '#comments') type='view-comments' text='View comments'}}
+	{{> link class='comments-button' href=(concat canonicalURL '#comments-anchor') type='view-comments' text='View comments'}}
```

Comment:
I'm just trusting this anchor is correct, but couldn't find an associated id with a quick search?

Reply:
Its legit https://www.ft.com/content/6b236a71-512e-4561-a73c-b1d69b7f486b#comments-anchor

Reply:
Thanks for checking, here's proof that it works when I tested it, I'm scrolled down from pressing the link and screenshotting the element.

<img width="290" alt="Screenshot 2021-03-24 at 12 10 14" src="https://user-images.githubusercontent.com/2445413/112308292-e54f5a80-8c99-11eb-99fe-28b110c7e67c.png">